db.go

```diff
@@ -203,6 +205,42 @@ func (database *Database) query(w http.ResponseWriter, r *http.Request) {
 	respond(w, http.StatusOK, result)
 }
 
+func (database *Database) getByIds(w http.ResponseWriter, r *http.Request) {
+	conf, auth, err := middleware.Extract(r, true)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	col := ""
+
+	_, r.URL.Path = ShiftPath(r.URL.Path)
+	col, r.URL.Path = ShiftPath(r.URL.Path)
+
+	var body map[string][]string
```

Comment:
I wonder if it'd be simpler to use a `[]string` directly where the caller of this API post `["id1", "id2"], ...]` and no need to wrap this in in an object. What do you think?

Reply:
Fixed